### PR TITLE
Updating transfer.sh link

### PR DESCRIPTION
**Pull request by tacerus**
Removing the checklist as I am only issuing an update:

Replaced the transfer.sh link, as the original one is reported as "The service at https://transfersh.com is of unknown origin and reported as cloud malware.", as stated by the developer in the repository, and should hence not be published here (or anywhere else).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -789,7 +789,7 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 #### Command-line file upload
 
 - [Beauties](https://github.com/dsx/beauties) - Minimalist file sharing written in Go, to be used primarily from Unix shell (e.g. with curl). Can be built as a Debian package for easy install. `MIT` `Go`
-- [transfer.sh](https://transfer.sh) - Easy file sharing from the command line. ([Source Code](https://github.com/dutchcoders/transfer.sh)) `MIT` `Go`
+- [transfer.sh](https://github.com/dutchcoders/transfer.sh/) - Easy file sharing from the command line. ([Source Code](https://github.com/dutchcoders/transfer.sh)) `MIT` `Go`
```
Comment (n8225) [inline README.md:792]: Remove `([Source Code](https://github.com/dutchcoders/transfer.sh))` as it is a duplicate link.

**Review by n8225 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tacerus) [reply README.md:792]: Fair!

Comment (mzch) [reply README.md:792]: ? https://transfer.sh is official site (but https://transfersh.com is unofficial site, it's my instance). Why deleted?

Comment (nodiscc) [reply README.md:792]: I don't get the point of this pull request. The  transfer.sh entry is fine as it is https://github.com/awesome-selfhosted/awesome-selfhosted#command-line-file-upload

**Pull request closed without merging**